test/e2e_flags.go

```diff
@@ -27,22 +27,22 @@ var Flags EnvironmentFlags
 // EventingEnvironmentFlags holds the e2e flags needed only by the eventing repo.
 type EnvironmentFlags struct {
 	WorkloadIdentity     bool
-	PubsubServiceAccount string
+	SourceServiceAccount string
```

Comment:
Why we rename it to `SourceServiceAccount`? Is it supposed to be used only for Sources?

Reply:
Yes, it is. It is referring to the KSA for sources.

Reply:
But we also use it for Channel, right? Is the naming `SourceServiceAccount` kind of misleading?

Reply:
`pubsubServiceAccount` is used in e2e script for a GCP service account, `sourceServiceAccount` is used to differentiate that (the input is a KSA), and map to the      ```workloadIdentityMapping:
        K8S_SOURCE_SERVICE_ACCOUNT: PUBSUB-SERVICE-ACCOUNT```in the auth configMap. Do you have a better naming option for it? (different from pubsubServiceAccount, as it is already taken).

Reply:
Maybe you can use `pubsubKSA` for `KSA` and `pubsubGSA` for `GSA`

Reply:
I think I'd just put it ServiceAccountName? and maybe add a comment

Reply:
Sounds good, I'll change it to ServiceAccountName and add a comment to point out that it is a KSA